backend/src/zac/werkvoorraad/data.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import List
+
+from zac.accounts.models import AccessRequest
+from zac.activities.models import Activity
+from zgw.models import Zaak
+
+
+@dataclass
+class ActivityGroup:
+    activities: List[Activity]
+    zaak: Zaak
+    zaak_url: str
+
+
+@dataclass
+class AccessRequestGroup:
+    requesters: List[AccessRequest]
```

Comment:
```suggestion
    requests: List[AccessRequest]
```
or `access_requests` is a better name, see the serializer remarks